package.json

```diff
@@ -42,7 +42,7 @@
     "@bower_components/crypto": "anodynos/node2web_crypto#3.46.1",
     "@bower_components/international-phone-number": "mareczek/international-phone-number#^0.0.16",
     "@bower_components/matchmedia": "paulirish/matchMedia.js#~0.2.0",
-    "@ovh-ux/telecom-universe-components": "^1.3.0",
+    "@ovh-ux/telecom-universe-components": "git+https://github.com/ovh-ux/telecom-universe-components.git#feat/holidays",
```

Comment:
@Steffy29 you are now able to upgrade to the latest telecom-universe-components.

ref: https://github.com/ovh-ux/telecom-universe-components/blob/master/CHANGELOG.md#140-2018-12-31